policyengine_us/parameters/gov/states/ca/cdss/tanf/exempt.yaml

```diff
@@ -0,0 +1,14 @@
+description: An assistance receives the Exempt MAP in California CalWORKs when it receives one of the following benefits.
+values:
+  2023-08-24:
```

Comment:
@MaxGhenis how far back should we include the values for the TANF program? the references do not mention historic values